src/main/java/org/automation/utilities/WebdriverWaits.java

```diff
@@ -124,14 +122,66 @@ public static void fluentWait_ElementLocated(long waitTimeForTimeout, long waitT
         wait.until(ExpectedConditions.visibilityOfElementLocated(locator));
 
     }
-    
+
     public static void fluentWait_ElementIntactable(long waitTimeForTimeout, long waitTimeForPolling, By locator) {
```

Comment:
Improvement Suggestion: Handle TimeoutException and log errors if the element is not interactable.
Good Use of Fluent Wait: Provides flexibility for dynamic wait intervals.